setup.py

```diff
@@ -180,6 +180,7 @@
             'jupyter-notebook = notebook.notebookapp:main',
             'jupyter-nbextension = notebook.nbextensions:main',
             'jupyter-serverextension = notebook.serverextensions:main',
+            'jupyter-bundler = notebook.bundlerextensions:main',
```

Comment:
Subcommands of notebook go through the earlier traitlets.config subcommand system. See the `subcommands` attribute on NotebookApp, where there should already be at least one example.

I'm not sure that it should change, but `bundler` seemed a little generic to be top-level, so I thought I'd ask.